pom.xml

```diff
@@ -47,12 +46,95 @@
             <artifactId>hadoop-client</artifactId>
             <version>2.2.0</version>
         </dependency>
+
+	<dependency>
+    	    <groupId>org.apache.hadoop</groupId>
+    	    <artifactId>hadoop-hdfs</artifactId>
+    	    <version>2.2.0</version>
```

Comment:
Later, we can use a variable (e.g. hadoopVersion) that specifies the Hadoop version and reuse them in pom.xml file